### DISCUSSION TITLE
Trigger a command by clicking on button

### DISCUSSION THREAD

**Discussion by Raduc4** [telegraf/telegraf#1462] [Q&A] [answered]
I have an inline keyboard (button) and when you click on it there should appear a list of options, again inline keyboard which live on /list command
How can i call a command on click?

**Comment by jsstrn** [comment, marked as answer]
As far as I know, you cannot do that. Instead, have them both call the same handler:

```js
const handleMyCommand = (ctx) => {...}

bot.command('my-command', handleMyCommand)

bot.action('my-action', handleMyCommand)
```

**Reply by winqdev** [reply]
what is that bruh

**Reply by Raduc4** [reply]
sorry, I don't remember why I have posted this here, very strange tbh

**Comment by mcfriend99** [comment]
I'm doing exactly this but for some weird reasons after a button action callback gets called once, it never gets called again. Any idea what can be responsible for this??
